services/storage/docker/healthcheck.py

```diff
@@ -20,18 +20,50 @@
 
 
 import os
+import subprocess
 import sys
 from urllib.request import urlopen
 
+from simcore_service_storage.main import ApplicationSettings
+
 SUCCESS, UNHEALTHY = 0, 1
 
 # Disabled if boots with debugger
-ok = os.environ.get("SC_BOOT_MODE", "").lower() == "debug"
+ok = os.getenv("SC_BOOT_MODE", "").lower() == "debug"
 
 # Queries host
 # pylint: disable=consider-using-with
+
+
+def _is_celery_worker_healthy():
+    app_settings = ApplicationSettings.create_from_envs()
+
+    assert app_settings.STORAGE_CELERY
+    broker_url = app_settings.STORAGE_CELERY.CELERY_BROKER.dsn
+
+    try:
+        result = subprocess.run(
+            [
+                "celery",
+                "-b",
+                broker_url,
+                "inspect",
+                "ping",
+                "-d",
+                "celery@" + os.getenv("HOSTNAME", ""),
```

Comment:
minor: Not sure if using the native `HOSTNAME` from the OS will work in *all* situation. Inside a docker container, it makes a slight difference for example whether you use `localhost`, `127.0.0.1` or `0.0.0.0` :shrug: If you are sure this works I am fine though :--)